### PR TITLE
update eds and other packages

### LINKED ISSUE #1138: update eds-core-react
there are some breaking changes

### PR DESCRIPTION
**Pull request by KristianKjerstad**
## What does this pull request change?
update packages, fix breaking changes from eds core react
## Why is this pull request needed?
update packages
## Issues related to this change:

closes #1129 
closes #1120
closes #1122 

closes #1138

### CODE REVIEW

File: web/packages/plugins/shared/common/src/components/Modal/CustomScrim.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Dialog, Icon, Scrim } from '@equinor/eds-core-react'
 import React from 'react'
 import styled from 'styled-components'
+import { isObject } from 'lodash'
```
Comment (soofstad) [inline web/packages/plugins/shared/common/src/components/Modal/CustomScrim.tsx:4]: Don't see this being used. Can we remove the "lodash" package?

**Review by soofstad — APPROVED** [review]

File: web/packages/plugins/shared/common/src/components/Modal/CustomScrim.tsx (hunk 1 above)
Comment (KristianKjerstad) [reply web/packages/plugins/shared/common/src/components/Modal/CustomScrim.tsx]: lodash seems to be used indirectly by other packages (not used in any of our package.json files)

File: web/packages/plugins/shared/common/src/components/NewEntityButton.tsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ export function NewEntityButton(props: {
             justifyContent: 'space-between',
           }}
         >
-          <Label label={'Folder for new entity'} />
+          <Label label={typeToCreate} />
```
Comment (soofstad) [inline web/packages/plugins/shared/common/src/components/NewEntityButton.tsx:55]: Why is this changed? The DestinationPicker should have a description on what you are picking for

**Review by soofstad — CHANGES_REQUESTED** [review]

**Comment by KristianKjerstad** [comment]
@soofstad waiting for approval on this

**Review by soofstad — APPROVED** [review]

**Pull request merged**